crates/cheatcodes/src/test/expect.rs

```diff
@@ -578,7 +579,9 @@ pub(crate) fn handle_expect_revert(
         }
     }
 
-    if actual_revert == expected_revert {
+    if actual_revert == expected_revert ||
+        (is_cheatcode && memchr::memmem::find(&actual_revert, expected_revert).is_some())
+    {
```

Comment:
@mattsse @klkvr FYI this changes `_expectCheatcodeRevert` to match on substring as well, to avoid changing all our tests

Reply:
this seems fine